docs/storybook-react/tailwind.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
maybe put a comment here to remind us to also import design-tokens settings as well?